outlines/models/transformers.py

```diff
@@ -72,13 +66,13 @@ def __init__(self, model_name: str, **kwargs):
 
     def encode(
         self, prompt: Union[str, List[str]], **kwargs
-    ) -> Tuple[NDArray[np.int64], NDArray[np.int64]]:
+    ) -> Tuple[torch.LongTensor, torch.LongTensor]:
         kwargs["padding"] = True
-        kwargs["return_tensors"] = "np"
+        kwargs["return_tensors"] = "pt"
         output = self.tokenizer(prompt, **kwargs)
         return output["input_ids"], output["attention_mask"]
```

Comment:
We must transfer the tokens onto the device somewhere. This code currently fails as input tokens are not on gpu. 
`return output["input_ids"].to(device), output["attention_mask"].to(device)` should fix it.